test/plugin/currencyPlugin/fixtures.json

```diff
@@ -41,6 +41,12 @@
         "nativeAmount": "1234567890000",
         "currencyCode": "BTC"
       }],
+      "uri address with amount and without prefix": ["1F1tAaz5x1HUXrCNLbtMDqcw6o5GNn4xqX?amount=12345.6789", {
```

Comment:
This should require a corresponding change to the testing code

Reply:
@paullinator as I understand - no, because tests are launched in this way:
```
describe(`parseUri for Wallet type ${WALLET_TYPE}`, function() {
    Object.keys(fixture.parseUri).forEach(test => {
      if (fixture.parseUri[test].length === 2) {
        it(test, async function() {
          const parsedUri = await tools.parseUri(fixture.parseUri[test][0])
          const expectedParsedUri = fixture.parseUri[test][1]
          assert.deepEqual(parsedUri, expectedParsedUri)
        })
      } else {
        it(test, function() {
          assert.throws(() => tools.parseUri(fixture.parseUri[test][0]))
        })
      }
    })
  })
```

also I check the tests and saw my case. only one question, should I add this tests for all currencies in fixtures.json?